### DISCUSSION TITLE
Access to the repository settings

### DISCUSSION THREAD

**Discussion by Salkimmich** [ccc-certifier-framework/certifier-framework-for-confidential-computing#237] [General]
Hello Certifier Framework Maintainers:

I'm reaching out to request access to the repository settings for the Certifier Framework. Many best practices have already been established within the project, but I am preparing for future updates and enhancements to further strengthen security, compliance, and collaboration efforts.

To effectively review and enhance the project's security, compliance, and collaboration efforts, I require ownership or administrative privileges within the repository. This level of access will enable me to conduct a thorough review, identify areas for improvement, and propose enhancements that align with the project's objectives. 

Granting me access to the repository settings will enable me to review the existing configurations, identify any areas for improvement, and implement additional best practices and other measures to enhance the development workflow and project governance. Best practices can be highly specific to each project's unique setup and requirements, so our approach will be flexible and tailored to accommodate the Certifier Framework's specific needs.

Following review, any proposed changes to the process will be discussed transparently with the project maintainers and contributors. The standard PR and Issue process will be used to ensure full visibility and collaboration on all recommendations. 

I look forward to collaborating with you for future updates!

Thanks,
Sal 
CCC Technical Community Architect

**Comment by jlmucb** [comment]
OK will do

**Comment by gapisback** [comment]
hello, @Salkimmich , Good morning! I just came upon this request. I have worked on this repo alongside @jlmucb (but am no longer with VMware now).

I am particularly interested in your ask for: " _implement additional best practices and other measures to enhance the development workflow_".

I have implemented a whole bunch of productization, code-tightening, build-and-test for sample programs, tests, test-dev, pytests, Python bindings (for python app-dev) and CI-jobs. So am somewhat vested in keeping track of the kind(s) of improvements you are suggesting.

Although I am not officially working on CC/CF, I am interested to keep track of how this matures.

Request to both of you to kindly keep me in the loop, so I can help & contribute wherever possible.
